api/v1/policy_types.go

```diff
@@ -108,13 +108,26 @@ type DetailsPerTemplate struct {
 	History         []ComplianceHistory `json:"history,omitempty"`
 }
 
+// ReplicatedDetailsPerTemplate defines the replicated policy compliance details and history
+type ReplicatedDetailsPerTemplate struct {
```

Comment:
Could you please move this to `api/v1beta1/policyautomation_types.go` since this replicated policies actually do use `DetailsPerTemplate`?